scripts/update/0.6-0.6.1/cluster.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
Shouldn't the directory be named `0.6.0-0.6.1`?

Reply:
When we released 0.6 version , we didnt say it as v0.6.0 :thinking:

Reply:
What do you mean? The tag is v0.6.0.

Reply:
updated the directory name.